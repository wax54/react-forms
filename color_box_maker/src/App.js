import React from "react";
import BoxList from './BoxList';
import "./App.css";

/** Simple app that makes a box list which can make colored boxes of various sizes */

function App() {
  return (
    <div className="App">
      <BoxList />
    </div>
  );
}

export default App;
