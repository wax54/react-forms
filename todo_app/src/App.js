import React from "react";
import TodoList from './TodoList';
import "./App.css";

/** Simple app that just shows the LightsOut game. */

function App() {
  return (
    <div className="App">
      <TodoList />
    </div>
  );
}

export default App;
