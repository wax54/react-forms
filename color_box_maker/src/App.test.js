import React from 'react';
import { render, fireEvent } from "@testing-library/react";
import App from './App';

it("Doesn't Blow up", () => {
    render(<App />);
});

it('should match previous snapshot', function () {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
});
