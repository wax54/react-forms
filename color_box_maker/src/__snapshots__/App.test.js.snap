// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match previous snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <div
      class="BoxList"
    >
      <div
        class="Box"
        style="height: 100px; width: 100px; background-color: black;"
      />
      <button>
        X
      </button>
      <form>
        <label
          for="height-input"
        >
          Height: 
        </label>
        <input
          id="height-input"
          min="0"
          name="height"
          required=""
          type="number"
          value=""
        />
        <label
          for="width-input"
        >
          Width: 
        </label>
        <input
          id="width-input"
          min="0"
          name="width"
          required=""
          type="number"
          value=""
        />
        <label
          for="color-input"
        >
          Color: 
        </label>
        <input
          id="color-input"
          name="color"
          required=""
          type="text"
          value=""
        />
        <button>
          Make me a Box!
        </button>
      </form>
    </div>
  </div>
</DocumentFragment>
`;
